servers/audio/audio_stream.cpp

```diff
@@ -362,13 +357,13 @@ int AudioStreamPlaybackMicrophone::_mix_internal(AudioFrame *p_buffer, int p_fra
 	}
 
 #ifdef DEBUG_ENABLED
-	if (input_ofs > input_position && (int)(input_ofs - input_position) < (p_frames * 2)) {
-		print_verbose(String(get_class_name()) + " buffer underrun: input_position=" + itos(input_position) + " input_ofs=" + itos(input_ofs) + " input_size=" + itos(input_size));
+	if (mixed_frames != p_frames) {
+		ERR_PRINT(vformat("Buffer underrun: size_position.size = %d, input_ofs = %d, buf.size() = %d.", size_position.size, input_ofs, buf.size()));
```

Comment:
It was useful for debugging, decided to keep it
Copied from: https://github.com/godotengine/godot/pull/92969#discussion_r1633016298.